lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
@cjbj yes we got that refcount thing already, what we are talking about here is a different thing. I'll try to give some more details.
Until now we were always binding a value that corresponds to a `CLOB` type column in the database to OCI's `SQLT_CHR` binding type (without using a descriptor) and it perfectly worked in PHP5.
Now in PHP7 using `SQLT_CHR` for binding `CLOB` type column values does not work anymore and requires a descriptor + binding type `OCI_B_CLOB`.
I thought you might have a bit more background information why this is the case and if this is intended.